packages/cli/README-source.md

```diff
@@ -529,16 +529,52 @@ You can find more details on the definition for each by looking at the [Trigger
 > To add a trigger, search, or create to an existing integration, run `zapier scaffold [trigger|search|create] [noun]` to create the necessary files to your project. For example, `zapier scaffold trigger post` will create a new trigger called "New Post".
 ### Return Types
 
-Each of the 3 types of function expects a certain type of object. As of core v1.0.11, there are automated checks to let you know when you're trying to pass the wrong type back. For reference, each expects:
+Each of the 3 types of function should return a certain data type for use by the platform. As of core v1.0.11, there are automated checks to let you know when you're trying to pass the wrong type back. For reference, each expects:
```

Comment:
While you were there, can we remove the reference of v1.0.11 as it's ancient and we can assume no one is using platform-core older than v1.0.11?

```suggestion
Each of the 3 types of function should return a certain data type for use by the platform. There are automated checks to let you know when you're trying to pass the wrong type back. For reference, each expects:
```

Reply:
Absolutely! That was actually so old I wasn't even sure if it was referring to the same version set. Thanks for the catch!